submissions/oluwatosinOke/database/init.sql

```diff
@@ -0,0 +1,135 @@
+-- Create database
+CREATE DATABASE my_favorite_online_store;
+USE my_favorite_online_store;
+
+-- Create table named categories
+CREATE TABLE categories (
+id INT NOT NULL AUTO_INCREMENT,
+category_name VARCHAR(30) NOT NULL,
+PRIMARY KEY (id)
+);
+
+-- Insert values into table named categories
+INSERT INTO categories (category_name)
+VALUES ('Phones'),
+('Phone Accessories'),
+('Computers'),
+('Computer Accessories');
+
+-- Create table named admins
+CREATE TABLE admins (
+id INT NOT NULL AUTO_INCREMENT,
+NAME VARCHAR(30) NOT NULL,
```

Comment:
kindly use all lowercase for the column names. Same goes for others

Reply:
Requested changes have been effected.